### PR TITLE
Device Trust: TPM Enrollment support EKCerts

### PR DESCRIPTION
**Pull request by strideynet**
Related https://github.com/gravitational/teleport.e/pull/1519

As per https://github.com/gravitational/teleport.e/pull/911/files

Contains two "sets" of changes:
- Adds the appropriate fields to the config file struct/protos - and the logic to load/validate them
- Adds support to the native implementation for Windows TPM support to send an EKCert

### CODE REVIEW

File: lib/config/fileconf.go [hunk 1]
```diff
@@ -1338,6 +1338,16 @@ type DeviceTrust struct {
 	Mode string `yaml:"mode,omitempty"`
 	// AutoEnroll is the toggle for the device auto-enroll feature.
 	AutoEnroll string `yaml:"auto_enroll,omitempty"`
+	// Allow list of EKCert CAs. These may be specified as a PEM encoded
```
Comment (codingllama) [inline lib/config/fileconf.go:1341]: nit: "EKCertAllowedCAs is ..."

File: lib/config/testdata/tpm_ekcert_ca.pem [hunk 2]
```diff
@@ -0,0 +1,24 @@
+This is a genuine PEM for STM TPM EK Intermediate CA 06
```
Comment (codingllama) [inline lib/config/testdata/tpm_ekcert_ca.pem:1]: Do we have a download URL to add here too?

**Review by codingllama — APPROVED** [review]

File: lib/config/fileconf.go (hunk 1 above)
Comment (strideynet) [reply lib/config/fileconf.go]: 3ce2248841

**Review by ryanclark — APPROVED** [review]

**Review by ryanclark — APPROVED** [review]

File: lib/config/testdata/tpm_ekcert_ca.pem (hunk 2 above)
Comment (strideynet) [reply lib/config/testdata/tpm_ekcert_ca.pem:1]: 9faeb891fe

**Pull request merged**